python/CMakeLists.txt

```diff
@@ -6,11 +6,11 @@ else()
     find_package(pybind11)
 endif()
 
-add_library(sonata_python MODULE bindings.cpp)
+pybind11_add_module(sonata_python SYSTEM bindings.cpp)
+
 set_target_properties(sonata_python PROPERTIES
     PREFIX ""
-    OUTPUT_NAME "_libsonata_python"
-    LIBRARY_OUTPUT_DIRECTORY ${SONATA_PYTHON_BUILD}
+    OUTPUT_NAME "libsonata"
```

Comment:
doesn't it clash with the C++ library name?

Reply:
The actual name ends up being: libsonata.cpython-36m-x86_64-linux-gnu.so

Reply:
interesting :)
I figure we still need to explicitly specify PREFIX, right?

Reply:
It seems than not

Reply:
Don't know, the prefix might be platform-specific.

Reply:
(even if it is not, I would expect it that by default `PREFIX` is `lib`. Weird. ^^)